src/rocshmem_gpu.cpp

```diff
@@ -283,23 +283,24 @@ __device__ Context *get_internal_ctx(rocshmem_ctx_t ctx) {
   return reinterpret_cast<Context *>(ctx.ctx_opaque);
 }
 
-__device__ int rocshmem_wg_ctx_create(long option, rocshmem_ctx_t *ctx) {
-  GPU_DPRINTF("Function: rocshmem_ctx_create\n");
+__device__ int rocshmem_wg_ctx_create(long options, rocshmem_ctx_t *ctx) {
+  GPU_DPRINTF("Function: rocshmem_ctx_create (options=%ld)\n", options);
```

Comment:
Not introduced by this PR but

```suggestion
  GPU_DPRINTF("Function: rocshmem_wg_ctx_create (options=%ld)\n", options);
```